.gitignore

```diff
@@ -2,7 +2,7 @@
 /out/
 
 # npm deps
-node_modules
+/third_party/node_modules
```

Comment:
please don't change this line - we might have multiple `node_modules` in near future

Reply:
Thanks for the comment! I updated the line.